requirements-min.txt

```diff
@@ -9,3 +9,4 @@ pystac[validation] == 1.2
 rasterio == 1.2.9
 requests == 2.20
 stac-check == 1.1.3
+colorama == 0.4.4
```

Comment:
So this `requirements-min.txt` file is only used for CI (which makes me wonder if we should move it into the `.github` folder 🤔). To specify as a requirements, this needs to be added to `setup.cfg`.

Also, since this is a Windows-only requirement, we can guard the install w/ something like this: `colorma >= 0.4; sys_platform = 'win32'` (or something, see https://stackoverflow.com/a/35614580/732529).